starter-code/scripts/article.js

```diff
@@ -1,30 +1,40 @@
 var articles = [];
 
 function Article (options) {
-/* TODO: This is our Model constructor. It will take in
+/* TODONE: This is our Model constructor. It will take in
    our source data from blogArticles and instantiate a
    new Object according to this new definition. options is
    a placeholder for the object that will ultimately be
    passed in. Use all of the properties in blogArticles
    to populate the new Article data that we'll use.  */
-   this.title = options.title;
+  this.title = options.title;
+  this.category = options.category;
+  this.author = options.author;
+  this.authorUrl = options.authorUrl;
+  this.publishedOn = options.publishedOn;
+  this.body = options.body;
 };
 
 /* This toHtml function is one that we are creating, and it is
    being attached to the prototype. Why? */
 Article.prototype.toHtml = function() {
   var $newArticle = $('article.template').clone();
   $newArticle.attr('data-category', this.category);
-  /* TODO: We also need to fill in:
+  /* TODONE?: We also need to fill in:
   1. author name
   2. author url
   3. article title
   4. article body
   5. publication*/
+  $newArticle.find('a').text(this.author);
+  $newArticle.find('a[href]').attr('href', this.authorUrl);
```

Comment:
Great job! This can also be written as $newArticle.find('a').attr('href', this.authorUrl); they both work equally well, so you can use either!